fluent.syntax/docs/usage.rst

```diff
@@ -0,0 +1,107 @@
+Using fluent.syntax
+===================
+
+The ``fluent.syntax`` package provides a parser, a serializer, and libraries
+for analysis and processing of Fluent files.
+
+Parsing
+-------
+
+To parse a full resource, you can use the :py:func:`fluent.syntax.parse`
+shorthand:
+
+.. code-block:: python
+
+   from fluent.syntax import parse
+   resource = parse("""
+   ### Fluent resource comment
+
+   first = creating a { $thing }
+   second = more content
+   """)
+
+To parse a single :py:class:`fluent.syntax.ast.Message` or :py:class:`fluent.syntax.ast.Term`, use
+:py:meth:`fluent.syntax.parser.FluentParser.parse_entry`:
+
+.. code-block:: python
+
+   from fluent.syntax.parser import FluentParser
+   parser = FluentParser()
+   key_message = parser.parse_entry("""
+   ### Fluent resource comment
+
+   key = value
+   """)
+
+Serialization
+-------------
+
+To create Fluent syntax from AST objects, use :py:func:`fluent.syntax.serialize` or
+:py:class:`fluent.syntax.serializer.FluentSerializer`.
+
+.. code-block:: python
+
+   from fluent.syntax import serialize
+   from fluent.syntax.serialzier import FluentSerializer
+   serialize(resource)
+   serializer = FluentSerializer()
+   serializer.serialize_entry(key_message)
```

Comment:
`serialize_entry` takes a full entry node, e.g. `resource.body[0]`. How about also using `serializer.serialize(resource)` here instead, to show that it's an alternative to the `serialize` helper?

Reply:
Or maybe show both?

```py
serializer = FluentSerializer()
serializer.serialize(resource)
serializer.serialize_entry(message)
```